superset/databases/api.py

```diff
@@ -1447,6 +1447,7 @@ def validate_parameters(self) -> FlaskResponse:
     @expose("/<int:pk>/ssh_tunnel/", methods=("DELETE",))
     @protect()
     @statsd_metrics
+    @deprecated(deprecated_in="4.0")
```

Comment:
@geido doesn't this mean that we'll be removing this endpoint in 4.0?

Reply:
No. It means that the endpoint was deprecated in 4.0 to be removed in a future version. Possibly 5.0.

Reply:
Yeah the idea is indeed to remove it in a future version